README.md

```
@@ -17,6 +17,18 @@ The language server features include:
-Otherwise, you can download it from this repo's [releases](https://github.com/formancehq/numscript/releases)
+### Installation
+
+You can install the `numscript` cli with one of the following ways:
+
+**Using curl**
+
+```sh
+curl -sSf https://raw.githubusercontent.com/formancehq/numscript/refs/heads/main/install.sh | bash
+```
+
+**Using golang toolchain**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)